includes/admin/customers/customers.php

```diff
@@ -800,6 +800,7 @@ function edd_customers_view( $customer = null ) {
 						<div class="add-customer-email-wrapper">
 							<input type="hidden" name="customer-id" value="<?php echo esc_attr( $customer->id ); ?>" />
 							<?php wp_nonce_field( 'edd-add-customer-email', 'add_email_nonce', false, true ); ?>
+							<label class = "hidden" for="additional-email">Email Address</label>
 							<input type="email" name="additional-email" value="" placeholder="<?php esc_html_e( 'Email Address', 'easy-digital-downloads' ); ?>" />&nbsp;
```

Comment:
@LisaCee A `label` is linked to an `input` via matching the label's `for` attribute to the input's `id` attribute. This is because an input can have match name values, such as a list of checkboxes:

```
<input name="foo[]" value="1" />
<input name="foo[]" value="2" />
<input name="foo[]" value="3" />
```

The input below this label needs to have an ID for the label's `for` attribute to be correct.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label